boards/atmel/sam/sam_e70_xplained/sam_e70_xplained-common.dtsi

```diff
@@ -238,3 +239,16 @@ zephyr_udc0: &usbhs {
 		pinctrl-names = "default";
 	};
 };
+
+&hsmci {
+	status = "okay";
+
+	pinctrl-names = "default";
+	pinctrl-0 = <&hsmci_default>;
+	cd-gpios = <&pioc 16 GPIO_ACTIVE_LOW>;
```

Comment:
please, consider to add this to `samv71_xult` board too. All configs are the same except the `cd-gpio` which should be
`	cd-gpios = <&piod 18 GPIO_ACTIVE_LOW>;`